apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -64,9 +97,53 @@ export default class LevelDetailsDialog extends Component {
     }
   }
 
+  handleBubbleChoiceBubbleClick = level => {
+    const previousSelected = this.state.selectedLevel;
```

Comment:
I had some trouble figuring out how to get the selected bubble to be highlighted and ended up adding a `highlighted` property to levels. This gets messy though because that means we need to do deep copies of the state and we need to either set the `highlighted` property on the script level or one of the sublevels. I'm definitely open to ideas on how to handle this better!